src/Sarif.Driver.UnitTests/TestAnalyzeCommand.cs

```diff
@@ -26,5 +26,14 @@ public class TestAnalyzeCommand : AnalyzeCommandBase<TestAnalysisContext, TestAn
             context.Options = options;
             return context;
         }
+
+        protected override void ValidateOptions(TestAnalysisContext context, TestAnalyzeOptions options)
+        {
```

Comment:
OK, so the implicit proposal here is that someone overriding ValidateOptions should produce an ExitApplicationException in the event that options don't validate. Any other unhandled exception would be regarded as a tool error. Yes?